dbt/include/databricks/macros/relations/tblproperties.sql

```diff
@@ -15,6 +19,10 @@
 
 {% macro apply_tblproperties(relation, tblproperties) -%}
   {% if tblproperties %}
+    {%- set is_iceberg = adapter.check_iceberg(config.get('table_format'), config.get('file_format'), model.config.materialized) -%}
```

Comment:
Same as above, but this statement is used to alter tables constructed in other ways, such as via python models.